Source/Classes/Drawing/Truncation.swift

```diff
@@ -48,7 +48,7 @@ extension NantesLabel {
         // Walk across all the lines of truncation, replacing lines starting with our last line - the number of truncation token lines we have
         // the first line we replace, we'll truncate it, after that, we 100% replace lines of the original string with truncation lines
         for (index, tokenLine) in tokenLines.enumerated() {
-            let originalIndex = self.numberOfLines - tokenLines.count + index
+            let originalIndex = lines.count - tokenLines.count + index
```

Comment:
The settings when the app crashed were as follows.

- self.numberOfLines: `5`
- lines.count: `4`
- tokenLines.count: `2`

As a result, the originalIndex was out of range of lines and crashed at times other than the first replacement.
When there are many lines with only line breaks, the number of lines in `lines` may not match with `numberOfLines`, so the index is acquired from the number of lines in `lines`.